Source/WebCore/dom/DocumentMarker.h

```diff
@@ -126,6 +126,10 @@ class DocumentMarker : public CanMakeWeakPtr<DocumentMarker> {
     };
 #endif
 
+    struct TransparentContentData {
```

Comment:
Type: `Transparent`

Reply:
I'm not sure what you mean by this comment?

Reply:
I don't know what I was doing either.

Reply:
I think there is a typo in "Type" lol